_posts/2018-02-28-incremental-dexing.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: posts
+title: "Bazel 0.11"
```

Comment:
I think you should change the title here to something like: "Incremental Dexing"